### PR TITLE
Fix clone

### PR DESCRIPTION
**Pull request by davidx**
fixes clone problem state, consolidate into refresh_repo
allows for bin/console use without erroring on missing logger

### CODE REVIEW

**Comment by ooshlablu** [comment]
:+1:

File: lib/thumbs/pull_request_worker.rb [hunk 1]
```diff
@@ -176,7 +176,9 @@ def reviews
 
     def debug_message(message)
       log = Log4r::Logger['Thumbs']
-      log.debug("#{@repo} #{@pr.number} #{@pr.state} #{message}")
+      if log
```
Comment (paegun) [inline lib/thumbs/pull_request_worker.rb:179]: I'll submit a PR later for reducing the de-ref'ing of the logger and subsequent conditional. In short, good catch, here to not log if there's no logger.

Comment (davidx) [reply lib/thumbs/pull_request_worker.rb:179]: excellent, thanks!

**Comment by paegun** [comment]
+1 f3bb432

**Pull request merged**